ccdaservice/oe-blue-button-util/lib/jsonpath/index.js

```diff
@@ -113,7 +113,7 @@ var processOptions = function (opts) {
     return {
         resultType: (opts && opts.resultType && opts.resultType.toLowerCase()) || 'value',
         flatten: (opts && opts.flatten) || false,
-        wrap: (opts && opts.hasOwnProperty('wrap')) ? opts.wrap : null,
+        wrap: (opts && Object.prototype.hasOwnProperty.call(opts, 'wrap')) ? opts.wrap : null,
```

Comment:
Is the concern here with Object.prototype just to ensure that someone hasn't overridden the hasOwnProperty method?

Reply:
That's part of the reason. The other part is because the object may not have access to this function, even though it comes from Object.prototype. This happens when you create an object using `Object.create(null)`. Therefore, the operation is deemed unsafe and the linter throws an error.

If you'd like more details you can read [here](https://eslint.org/docs/latest/rules/no-prototype-builtins).

Reply:
Sounds good.  Thanks for the clarification.